orbitx/flight_gui.py

```diff
@@ -88,7 +90,7 @@ def callback(*_):
 
         assert len(physical_state_to_draw.entities) >= 1
         self._last_physical_state = physical_state_to_draw
-        # Unessary code as these are reset through _set_X?
+        # Unnecessary code as these are reset through _set_X?
```

Comment:
Patrick can I delete the below three lines? as if you see line 97,98,99 these are set through functions you defined.

Reply:
You can, but if the DEFAULT_REFERENCE and other defaults don't exist in the save file we're using, removing these lines will make the references be None instead of the first entity. Deleting the code is good, since it's 1) less code and 2) a noisier (if more confusing) failure when the defaults don't exist.